### DISCUSSION TITLE
Help! I am translating into Japanese. Please tell me the exact meaning in English.

### DISCUSSION THREAD

**Discussion by mi-ya1987** [kwsch/PKHeX#3793] [Q&A]
[PKHeX.Core](https://github.com/kwsch/PKHeX/tree/master/PKHeX.Core)/[Resources](https://github.com/kwsch/PKHeX/tree/master/PKHeX.Core/Resources)/[text](https://github.com/kwsch/PKHeX/tree/master/PKHeX.Core/Resources/text)/[program](https://github.com/kwsch/PKHeX/tree/master/PKHeX.Core/Resources/text/program)/MessageStrings_en.txt
****
1      MsgProgramRestart = Please restart _**the program**_.

"the program" means only "PKHeX"? Or does it also include "Plugins"?
The expression in Japanese changes depending on whether it is included or not.

**Comment by Lusamine** [comment]
It would mean that you close PKHeX and open it again. Any plugins would be automatically loaded. It would not make sense to only restart plugins.

**Reply by mi-ya1987** [reply]
Is it okay to say "Restart PKHeX" when writing in Japanese?
